grakn-graph/src/test/java/ai/grakn/graph/internal/GraknTinkerGraphTest.java

```diff
@@ -80,8 +79,7 @@ public void testTestThreadLocal(){
 
         for(int i = 0; i < 100; i ++){
             futures.add(pool.submit(() -> {
-                GraknGraph innerTranscation = graknGraph;
-                innerTranscation.open();
+                GraknGraph innerTranscation = Grakn.factory(Grakn.IN_MEMORY, graknGraph.getKeyspace()).getGraph();
```

Comment:
You have to open graphs in different threads but you do so via the factory. So now you don't pass a graph into a different thread. You pass the factory.